wiki/Music_theory/Downbeat/de.md

```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+tags:
+  - downbeats
+  - first beat
+  - erste Note
+---
+
+# Downbeat
+
+Der **Downbeat** ist die erste Note in einem Takt. Er ist auf der Zeitleiste des [Beatmap-Editors](/wiki/Client/Beatmap_editor) als große weiße Striche erkennbar. Sein Name ist von der Abwärtsbewegung des Dirigentenstabs bei der ersten Note jedes Takts abgeleitet. In der Regel ist er im rhythmischen Zyklus am meisten betont.
```

Comment:
I know the English version uses the plural here, but the translation doesn't sound great.

```suggestion
Der **Downbeat** ist die erste Note in einem Takt. Er ist auf der Zeitleiste des [Beatmap-Editors](/wiki/Client/Beatmap_editor) als großer weißer Strich erkennbar. Sein Name ist von der Abwärtsbewegung des Dirigentenstabs bei der ersten Note jedes Takts abgeleitet. In der Regel ist er im rhythmischen Zyklus am meisten betont.
```